airgun/entities/subscription.py

```diff
@@ -13,14 +13,17 @@
 
 
 class SubscriptionEntity(BaseEntity):
-    def _wait_for_process_to_finish(self, name, has_manifest=False, timeout=600):
+    def _wait_for_process_to_finish(self, name, has_manifest=False, timeout=600,
+                                    ignore_error_messages=None):
         """Helper ensuring that task (upload / delete manifest / subscription)
         has finished. Run after action invoking task to leave Satellite
         in usable state.
         Currently waits for three events. Since page is written asynchronously,
         they can happen in any order.
         :param name: Name of running task
         :param has_manifest: Should manifest exist after task ended?
+        :param timeout: Waiting timeout
+        :param ignore_error_messages: Error messages to ignore
```

Comment:
Could you elaborate a bit what data structure is expected here? Maybe "List of strings with error messages to ignore; exact match will be used"?